finagle-thrift/src/main/ruby/README.md

```diff
@@ -0,0 +1,13 @@
+# Finable Thrift - Trace Ruby Thread Saftey
+
+## Synopsis
+Design decisions behind thread saftey in Trace
```

Comment:
I'm not sure this makes sense from a user perspective.  Most folks reading the README probably don't care terribly about the design decision, and probably care more about how to use it.  If you want to document the design decision, probably best to do that in the code.  Could we keep the README just focusing on what is a valid use of tracing?